Carmunication/Extensions/String+Extension.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  File.swift
+//  Carmunication
+//
+//  Created by 김동현 on 2023/10/17.
+//
+
+import Foundation
+
+extension String {
+
+    static func removeCountryAndPostalCode(from subtitle: String) -> String {
+        var modifiedSubtitle = subtitle
+        modifiedSubtitle = modifiedSubtitle.replacingOccurrences(of: "대한민국", with: "")
```

Comment:
```suggestion
        var modifiedSubtitle = subtitle.replacingOccurrences(of: "대한민국", with: "")
```
이부분 이렇게 작성해도 될 것 같은데 혹시 이유가 있을까용?